src/plugins/intel_cpu/tests/functional/shared_tests_instances/subgraph_tests/rotary_pos_emb.cpp

```diff
@@ -79,5 +79,12 @@ INSTANTIATE_TEST_SUITE_P(smoke_RoPETestQwenVL,
                             ::testing::ValuesIn(vit_param)),
                          RoPETestQwenVL::getTestCaseName);
 
+INSTANTIATE_TEST_SUITE_P(smoke_RoPETestChatGLM,
+                         RoPETestChatGLMHF,
```

Comment:
[buildROPE_ChatGLMHF   ](https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/shared/src/subgraph/rotary_pos_emb.cpp#L1345)in "RoPETestChatGLMHF" seems not contain new pattern, can we extend?